acceptance/trivial/test/integration/chef-current-install/inspec/chef_client_spec.rb

```diff
@@ -0,0 +1,5 @@
+chef_version = ENV["KITCHEN_CHEF_VERSION"].split("+")[0]
```

Comment:
you'll want to rename the `chef-current-install` dir to `chefdk-current-install` as well